### PR TITLE
chore: Enhance title & description for Certificates on settings page

### LINKED ISSUE #7560: Better title and description for 'Certificates' on settings page
# Summary

On `/settings` page, one of the cards has the title 'Certificates' while in fact the link it points to does not contain any certificates. 
image

# Motivation

This could be misleading as SSH keys are also configured in `/settings/certs`. SSH keys are not considered certificates.

# Proposal

1. Change the title 'Certificates' to 'Certificates and known hosts'.
2. Change the description under that title accordingly.
3. Change the title of the page linked from the card.

### PR DESCRIPTION
**Pull request by aimanfatima**
Signed-off-by: Aiman Fatima <aimanfatimadl@gmail.com>
Closes #7560

Local testing - 
image
image

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/settings/components/settings-overview/settings-overview.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ const settings = [
         path: './repos'
     },
     {
-        title: 'Certificates',
-        description: 'Configure certificates for connecting Git repositories',
+        title: 'Certificates and known hosts',
```
Comment (saumeya) [inline ui/src/app/settings/components/settings-overview/settings-overview.tsx:16]: Can this also be 'Repository certificate and known hosts'. Just to make it consistent with the other settings.

**Review by saumeya — CHANGES_REQUESTED** [review]
Thanks for your contribution @aimanfatima, added a small comment.

File: ui/src/app/settings/components/settings-overview/settings-overview.tsx (hunk 1 above)
Comment (aimanfatima) [reply ui/src/app/settings/components/settings-overview/settings-overview.tsx]: @saumeya made the change. 
please note - I changed both title and description to Repository certificates

**Review by saumeya — APPROVED** [review]
Thanks! LGTM

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**